script/validate

```diff
@@ -23,3 +44,8 @@ for f in sys.argv[2:]:
     except jsonschema.exceptions.ValidationError as err:
         print(f"Validation error in '{f}': {err.message}")
         sys.exit(1)
+
+duplicates = [u for (u, c) in Counter(urns).most_common() if c > 1]
```

Comment:
Personally i would have used the unique() from Numpy but this is actually pretty neat, thanks for this inspiration :)

Reply:
The straightforward python to get to a unique set would be `set(urns)`, however, in this case I wanted to also get those urns that aren't unique (and I wanted to limit the printout to a reasonable number, 10 in this case).